make.js

```diff
@@ -20,9 +20,13 @@ const outputPath = {
 target.allDev = async () => {
   banner('Target: AllDev');
 
-  const backupPath = await patchPackageJsonVersion();
-  await target.package();
-  await revertToOriginalPackageJson(backupPath);
+  const backupPath = await patchPackageJsonUpdateVersion();
+
+  try {
+    await target.package();
+  } finally {
```

Comment:
@philsturgeon This should reduce even further the chance of ending up with a temporarily patched package.json in the workdir when something bad happens.